ipod-shuffle-4g.py

```diff
@@ -97,12 +97,19 @@ def group_tracks_by_id3_template(tracks, template):
     return sorted(grouped_tracks_dict.items())
 
 class Text2Speech(object):
-    valid_tts = {'pico2wave': True, 'RHVoice': True, 'espeak': True}
+    valid_tts = {'pico2wave': True, 'RHVoice': True, 'espeak': True, 'say': True}
 
     @staticmethod
     def check_support():
         voiceoverAvailable = False
 
+        # Check for say voiceover
```

Comment:
I recommend adding a note, that say is a mac specific tool

Reply:
I added to the comment and warning message.

Reply:
Thanks! I think it also makes sense to add it to the readme.

Reply:
I added a brief mention of "say" support to the readme. I couldn't find an up to date man page or relevant reference online and aside from mentioning that it's available, I can't think of what else to mention. Suggestions welcome :-)